Description: Remove the link to the external image in footer
 The link to http://ansible.cc/img/AnsibleWorks.png is on an external location
 and may not be accessible, or may be used for tracking users activity without
 their knowledge.
Author: Janos Guljas <janos@debian.org>
Last-Update: 2013-04-06

Index: ansible/docsite/latest/_themes/bootstrap/layout.html
===================================================================
--- ansible.orig/docsite/latest/_themes/bootstrap/layout.html	2013-07-13 22:17:29.000000000 +0200
+++ ansible/docsite/latest/_themes/bootstrap/layout.html	2013-07-13 22:17:29.000000000 +0200
@@ -119,7 +119,6 @@
 {%- block footer %}
 <footer class="footer">
 <center>
-<a href="http://ansibleworks.com"><img src="http://ansible.cc/img/AnsibleWorks.png" alt="AnsibleWorks"></a>
 <p>
     {%- if show_copyright %}
       {%- if hasdoc('copyright') %}
